### PR TITLE
Add support for custom headers in AutoImage

### PR DESCRIPTION
**Pull request by LeXuanKhanh**
## Please verify the following:

- [x] `yarn test` **jest** tests pass with new tests, if relevant
- [x] `yarn lint` **eslint** checks pass with new code, if relevant
- [x] `yarn format:check` **prettier** checks pass with new code, if relevant
- [x] `README.md` (or relevant documentation) has been updated with your changes

## Describe your PR
As the title, my project is currently using your boilerplate but AutoImage component is missing the feature, it would be great if the main repo supported it

### CODE REVIEW

File: boilerplate/app/components/AutoImage.tsx [hunk 1]
```diff
@@ -10,6 +10,9 @@ export interface AutoImageProps extends ImageProps {
    * How tall should the image be?
    */
   maxHeight?: number
+  headers?: {
+    [key: string]: string
+  }
```
Comment (frankcalise) [inline boilerplate/app/components/AutoImage.tsx:15]: Could you please document this to be consistent with the other props? Something mentioning that these are the headers for the request

**Review by frankcalise — CHANGES_REQUESTED** [review]
Thanks for this, left one comment.

Also, could you look into updating the documentation to match this change in https://github.com/infinitered/ignite/blob/master/docs/boilerplate/app/components/AutoImage.md?

**Comment by LeXuanKhanh** [comment]
> Thanks for this, left one comment.
> 
> Also, could you look into updating the documentation to match this change in https://github.com/infinitered/ignite/blob/master/docs/boilerplate/app/components/AutoImage.md?

I have updated the document, please check it., thanks

**Comment by LeXuanKhanh** [comment]
hi @frankcalise, any comment or response about my PR ? thanks

**Review by frankcalise — APPROVED** [review]
Thanks, sorry I've been busy. Appreciate you checking up on it!

**Pull request merged**